src/vs/editor/common/standaloneStrings.ts

```diff
@@ -18,16 +18,16 @@ export namespace AccessibilityHelpNLS {
 	export const auto_off = nls.localize("auto_off", "The application is configured to never be optimized for usage with a Screen Reader.");
 	export const screenReaderModeEnabled = nls.localize("screenReaderModeEnabled", "Screen Reader Optimized Mode enabled.");
 	export const screenReaderModeDisabled = nls.localize("screenReaderModeDisabled", "Screen Reader Optimized Mode disabled.");
-	export const tabFocusModeOnMsg = nls.localize("tabFocusModeOnMsg", "Pressing Tab in the current editor will move focus to the next focusable element. Toggle this behavior<keybinding:editor.action.toggleTabFocusMode>");
-	export const tabFocusModeOffMsg = nls.localize("tabFocusModeOffMsg", "Pressing Tab in the current editor will insert the tab character. Toggle this behavior<keybinding:editor.action.toggleTabFocusMode>");
-	export const stickScroll = nls.localize("stickScrollKb", "Focus Sticky Scroll<keybinding:editor.action.focusStickyScroll> to focus the currently nested scopes.");
+	export const tabFocusModeOnMsg = nls.localize("tabFocusModeOnMsg", "Pressing Tab in the current editor will move focus to the next focusable element. Toggle this behavior{0}", '<keybinding:editor.action.toggleTabFocusMode>');
```

Comment:
Feels like the localizers will need more information about what the {0} is, should you use `ILocalizeInfo.comments`? If you have these in brackets it might also look wrong in some languages and maybe they should get localized too?

Reply:
for most of the cases, I say the command name right before this variable. I don't think I need to do more for the very few cases when that's not true